workers/Worker-creation-happens-in-parallel.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<title>Test that creation of a "new Worker()" will occur in parallel to the main JS thread performing other computation, and can be joined with.</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+promise_test(t => {
+  return new Promise(resolve => {
+    let worker = new Worker("support/Worker-creation-happens-in-parallel.js");
+    let sab = new Uint8Array(new SharedArrayBuffer(16));
+    worker.postMessage(sab);
+    let end = performance.now() + 30*1000;
+    while(sab[0] != 1 && performance.now() < end) /*wait to join with the result*/;
```

Comment:
Would manually increasing the timeout value help?

(30 seconds of time to create a Worker does seem gratuitously excessively long time here, so that even a slow test runner should be able to cope?)

The reason that I manually added a timeout as opposed to having a direct `while(sab[0] != 1) /*nop*/;` loop here was that such a loop would result in the slow script dialog coming up if the test fails, and I wasn't sure if the test runner's timeout mechanism manages that. (at least when I was running `wpt serve` and manually navigated to the page, I wasn't getting such an automated timeout, but the page would hang without a manual timeout)